src/test/java/spring/boot/parser/service/CsvToBeanParserServiceTest.java

```diff
@@ -0,0 +1,87 @@
+package spring.boot.parser.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.univocity.parsers.common.record.Record;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class CsvToBeanParserServiceTest {
+    public static final String FILE_EMPTY = "src/main/resources/test_file_empty.csv";
+    private static final String FILE_NOT_EXIST = "src/main/resources/file_not_exist.csv";
+    private static final String FILE_DATA_OK = "src/main/resources/test_file_ok.csv";
```

Comment:
let's better move these files to `src/**test**/resources/` folder

Reply:
Done